_content/tour/fra/welcome.article

```diff
@@ -0,0 +1,207 @@
+
+** Ultimate Go Notebook
+
+Tout ce matériel et bien plus encore peuvent également être trouvés dans l'Ultimate Go Notebook
+
+- [[https://www.ardanlabs.com/ultimate-go-notebook/][Ultimate Go Notebook]]
+
+** Besoin d'Aide
+
+N'hésitez pas à contacter Bill Kennedy de Ardan Labs si vous rencontrez des problèmes
+ou des questions
+
+[[mailto:bill@ardanlabs.com][bill@ardanlabs.com]]
+
+** Notre expérience
+
+Nous avons enseigné Go à des milliers de développeurs dans le monde entier depuis 2014.
+Aucune autre entreprise ne le fait depuis plus longtemps et notre matériel s'est avéré aider
+les développeurs à démarrer 6 à 12 mois avant leur connaissance de Go. Nous savons
+de quelles connaissances les développeurs ont besoin pour être productifs et efficaces lors de l'écriture
+de logiciels en Go.
+
+Nos cours sont parfaits pour les développeurs de niveau intermédiaire qui ont au moins
+quelques mois ou années d'expérience dans l'écriture de code dans Go. Nos cours offrent une connaissance
+très approfondie du langage de programmation avec un accent important sur la mécanique du langage,
+les philosophies de conception et les directives. Nous nous concentrons sur l'enseignement de l'écriture
+de code en priorité sur la cohérence, l’intégrité, la lisibilité et la simplicité. Nous abordons beaucoup
+de choses sur « si la performance compte » en mettant l'accent sur la sympathie mécanique, la conception orientée
+données, le découplage et l'écriture/débogage de logiciels de production.
+
+** Notre Professeur
+
+William Kennedy est associé directeur chez Ardan Labs à Miami, en Floride. Ardan Labs
+est une société de développement et de formation performante qui accompagne les startups et
+entreprises Fortune 500. Il est également co-auteur du livre Go in Action, l'auteur
+du blog GoingGo.Net, et membre fondateur de GoBridge qui travaille à
+augmenter l’adoption de Go grâce à la diversité.
+
+*Video* *Formation*
+
+- [[https://education.ardanlabs.com/][Ultimate Go Video]]
+- [[http://youtube.ardanlabs.com/][Ardan Labs YouTube Channel]]
+
+*Blog*
+
+- [[https://www.ardanlabs.com/blog/][Ardan Labs Blog Technique]]
+
+*Podcasts*
+
+- [[https://ardanlabs.buzzsprout.com/][Ardan Labs Podcast: On Going Series]]
+
+** En savoir plus sur Go
+
+Go est un langage de programmation open source qui facilite la création de fichiers simples,
+logiciel fiable et efficace. Bien qu'il emprunte des idées à des langages existants,
+il a une nature unique et simple qui rend les programmes Go différents par leur caractère
+programmes écrits dans d’autres langues. Il équilibre les capacités d'un niveau bas
+langage système avec certaines fonctionnalités de haut niveau que vous voyez dans les langues modernes aujourd'hui.
+Cela crée un environnement de programmation qui vous permet d'être incroyablement productif,
+performant et totalement en contrôle; dans Go, vous pouvez écrire moins de code et faire bien plus.
+
```

Comment:
(This paragraph was missing)
Go est la fusion de la performance et de la productivité dans un langage que les développeurs de logiciels peuvent apprendre, utiliser et comprendre. Go n'est pas le langage C, mais il nous permet de bénéficier des avantages du langage C et des avantages des langages de programmation de plus haut niveau.